include/arizona_assert.hrl

```diff
@@ -0,0 +1,9 @@
+-include_lib("stdlib/include/assert.hrl").
+
+-define(ARIZONA_ASSERT_BODY(Pattern, Resp), (
+    ?assertNotEqual(nomatch, string:find(element(3, Resp), Pattern))
+)).
+
+-define(ARIZONA_ASSERT_STATUS(Status, Resp), (
+    ?assertEqual(Status, element(2, element(1, Resp)))
+)).
```

Comment:
These macros are "asserting" `httpc` requests, so maybe they could be renamed to `ARIZONA_ASSERT_REQ_*`.
Also, I suggest preserving them semantically in the assert macros, like `?assertArizonaBody` and `?assertArizonaStatus` (or `?assertArizonaReqBody` and `?assertArizonaReqStatus`).
I am uncertain if these would be truly beneficial for the consumer. I devised them while attempting to simplify Arizona app tests.

Reply:
So... in the description what I mentioned was that you could have less magic if you chose something like:

```erlang
-module(arizona_test).

-export([is_body/2]).

is_body(Got, Expected) ->
    string:find(element(3, Resp), Pattern) =/= nomatch.
```

with which you could write a test like

```erlang
...
   ?assert(arizona_test:is_body(Got, Expected)).
```

Reply:
Excellent, much better.

Reply:
... but we can create those functions when we reorganize the code, in any case. For the time being I think we're better than we were yesterday :)